docs: adicionado seção para tags, na documentação (#111)
levxyca/diciotech: CONTRIBUTING.md

ou

```json
{
   "title": "Termo técnico",
   "description": "Explicação sobre o termo",
   "content": {
        "code": "Código de exemplo"
   },
   "tags": ["Tag1", "Tag2"]
}
```

> **Observação:** caso a sua contribuição não se alinhe com qualquer uma das tags disponíveis, listadas na seção seguinte, verifique a seção [Adicionando uma nova tag no Diciotech](#adicionando-uma-nova-tag-no-diciotech).

> Antes de abrir o pull request, algumas **regras** e **requisitos** devem ser seguidos para uma maior organização e estabilidade do Diciotech.

Reviewer: Acho que podemos alterar um pouco, eu não sei se faz sentido, mas ter os termos regras e requisitos talvez assuster pessoas que sejam iniciantes e já estejam com medo de contribuir. O que você acha?

```suggestion
Antes de abrir o pull request, algumas **boas práticas** devem ser seguidas para uma maior organização e estabilidade do Diciotech.

```

- Se o termo conter um código de exemplo, evite o escrever em uma linguagem de programação específica, faça isso **apenas** se realmente necessário;

> - Um termo, de forma obrigatória, deve ser vinculado a, no mínimo, uma tag;

Reviewer: ```suggestion
- Um termo, deve ser vinculado a, no mínimo, uma tag;

```

- Atribuir a um termo apenas as tags presentes nessa documentação, da exata mesma forma e escrita;

> - Não pode haver duplicações de tags em um mesmo termo. Exemplo: `"tags": ["Conceito", "Conceito"]`;

Reviewer: ```suggestion
- Evite duplicações de tags em um mesmo termo. Exemplo: `"tags": ["Conceito", "Conceito"]`;

```

> - As tags atribuídas a um termo devem fazer total sentido com o mesmo.

Reviewer: ```suggestion
- Tome a cuidado para que as tags atribuídas a um termo façam sentido com o mesmo.

```

### Adicionando uma nova tag no Diciotech

> Você pode contribuir com o projeto propondo uma nova tag e a apresentando, como sugestão, por meio de uma [issue](https://github.com/levxyca/diciotech/issues) para que se possa abrir uma discussão a respeito, para uma futura adição através de pull request. As tags disponíveis **sempre** devem estar presentes nessa documentação.

Reviewer: ```suggestion
Você pode contribuir com o projeto propondo uma nova tag e a apresentando, como sugestão, por meio de uma [issue](https://github.com/levxyca/diciotech/issues) para que se possa abrir uma discussão a respeito, para uma futura adição através de pull request. As tags disponíveis **sempre** vão estar presentes nessa documentação.

```

Tags disponíveis:

- `Back-end`;
- `Biblioteca`;
- `Conceito`;
- `Design`;
- `Ferramenta`;
- `Framework`;
- `Front-end`;
- `Mobile`;
- `Paradigma`;
- `Versionamento`.

> Antes de criar uma issue e abrir um PR, todas as tags devem seguir algumas regras e requisitos:

Reviewer: ```suggestion
Antes de criar uma issue e abrir um PR, todas as tags devem seguir algumas boas práticas:

```

- Todas as tags devem estar no singular;
- Atualmente, as tags são _case sensitivity_, então, por padrão, apenas a primeira letra da tag deve ser maiúscula. Exemplo: Back-end, Conceito, Paradigma;
- Para adicionar uma nova tag, primeiro deve criar uma issue e, caso as pessoas usuárias concordem, um pull request deve ser aberto, contendo a tag. A PR sendo aprovada, a nova tag poderá ser vinculada a um termo;
- As tags devem ser mais generalistas e categóricas, já que dispomos de um campo de pesquisa para uma busca mais específica e precisa. Exemplo de tags: Front-end, Design, Back-end;

> - Não devem existir tags de linguagens de programação, bibliotecas e frameworks. Exemplo: JavaScript, C#, React.js.

Reviewer: Acho que essa linha não precisa, visto que o tópico anterior fala sobre a tags serem generalistas e categóricas.
